src/verify/lines.ts

```diff
@@ -15,7 +15,9 @@
  */
 
 // Use classes here instead of interfaces because we want runtime type data
+/* tslint:disable:no-static-only-classes */
 export class Line { }
+/* tslint:enable:no-static-only-classes */
```

Comment:
I would prefer that an empty class is *not* counted as a static-only class, since IMO that doesn't fit with the spirit of the lint.